Helper/Cart.php

```diff
@@ -1028,7 +1028,7 @@ function ($item) use ($imageBlock, &$totalAmount, &$diff, $storeId) {
                 ///////////////////////////////////////////
                 // Get item attributes / product properties
                 ///////////////////////////////////////////
-                $item_options = $item->getProduct()->getTypeInstance(true)->getOrderOptions($item->getProduct());
+                $item_options =$_product->getTypeInstance(true)->getOrderOptions($_product);
```

Comment:
seems like `getTypeInstance` doesnt accept a param? or am i looking at the wrong m2

https://github.com/magento/magento2/blob/2b7761832406cb0875d337ef33401db36741c2c6/app/code/Magento/Catalog/Model/Product.php#L691

Reply:
Correct. Fixed.